cmake/deps.txt

```diff
@@ -0,0 +1,20 @@
+abseil_cpp;https://github.com/abseil/abseil-cpp/archive/refs/tags/20211102.0.zip;ce61532df974d00025b1220408ce1c900d81baf2
```

Comment:
how about using something like json? it could make it more obvious what the different fields are. e.g., I can't tell what the last value on each line is here

Reply:
Yes, we could. But the downside is we can't easily parse it in bash scripts. We may use jq, but it is a new dependency and I'm hesitated on whether it is worth.

Reply:
fair point about bash.
as another suggestion, how about using python instead? python has json parsing and it's easier to write cross platform code with it.

Reply:
That requires a much bigger change. For example, now we have [install_protobuf](https://github.com/microsoft/onnxruntime/blob/main/tools/ci_build/github/linux/docker/scripts/install_protobuf.sh). I want it to be able to get the protobuf downloading URL from a centralized place, so that it will be easier to upgrade protobuf version. We have many such bash scripts. Though it's possible to convert them to python file, it requires a lot of changes. And, we will also need to worry about python version.  The code may work with one python version but not another. And, does it need extra python dependencies? Bash doesn't have the problem.

Reply:
my main concern is that it is not obvious (from here) what the values represent. using JSON could be one way to improve it.

e.g.,
```json
[{"name": "abseil_cpp", "url": "https://github.com/abseil/abseil-cpp/archive/refs/tags/20211102.0.zip", "sha1": "ce61532df974d00025b1220408ce1c900d81baf2"}, ...]
```

some other format with more contextual information is fine too. alternatively, we can document it in some readme, but then one has to look somewhere else.

Reply:
I can add a header line there and add a document.

Reply:
Added a header row.